package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "root",
   "private": true,
-  "nextVersionBump": "patch",
+  "nextVersionBump": "minor",
```

Comment:
Making this change a `minor` bump here since it changes the default behavior of the search element substantially. Open to reconsidering this though, since technically there are no breaking changes that require code updates.